Audacia.Log.AspNetCore/HttpDependencyBodyCaptureTelemetryInitializer.cs

```diff
@@ -32,13 +33,23 @@ public class HttpDependencyBodyCaptureTelemetryInitializer : ITelemetryInitializ
     public int MaxDepth { get; set; } = 32;
 
     /// <summary>
-    /// Gets or sets a value indicating whether the logging of all data in the request body is disabled.
+    /// Gets or sets a value indicating whether the HTTP dependencies should be captured and logged.
     /// </summary>
-    public bool DisableBodyContent { get; set; }
+    public bool DisabledHttpTracking { get; set; }
```

Comment:
Don't think this is actually used? If it is needed, should be renamed `Disable` not `Disabled`. If it is not needed then can remove from `LogDepedencyFilterConfig` too?

Reply:
Updated.

Reply:
@Audacia-RhysSmith don't think anything has changed RE this?